document/CoreFormat.mdk

```diff
@@ -319,6 +326,15 @@ Analysis tools that are configurable to produce hashes with a variety of commonl
 will interoperate most easily with such systems.
 ~
 
+### `tags` property { #fileReference-tags }
+
+A `fileReference` object may contain a property named `tags`, whose value is an array of zero or more arbitrary strings.
```

Comment:
this should be an array or zero or more UNIQUE arbitrary strings. i.e., it is a set, no repeated elements. other than this change, LGTM, don't block on my re-review